Sources/web3swift/Tokens/ERC888/Web3+ERC888.swift

```diff
@@ -8,7 +8,7 @@
 
 import Foundation
 import BigInt
-
+import Core
 
 // MultiDimensional Token Standard
```

Comment:
Please, change to documentation and add link https://github.com/ethereum/EIPs/issues/888